back/prisma/data/resources.ts

```diff
@@ -2,8 +2,8 @@ import { Prisma } from '@prisma/client'
 import slugify from 'slugify'
 
 export const resources: Omit<
-  Prisma.ResourceCreateArgs['data'],
-  'userId'
+  Omit<Prisma.ResourceCreateArgs['data'], 'userId'>,
```

Comment:
```suggestion
  Omit<Prisma.ResourceCreateArgs['data'], 'userId' | 
```
https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys